tests/bcr/pkg/pkg_test.go

```diff
@@ -37,10 +34,6 @@ func TestBuildFileGeneration(t *testing.T) {
 	yamltemplate.HTMLEscapeString("<b>foo</b>")
 }
 
-func TestGeneratedFilesPreferredOverProtos(t *testing.T) {
```

Comment:
Same here, I think that we should try to keep the existing test coverage around.